app/models/rbac.rb

```diff
@@ -248,7 +248,33 @@ def self.find_targets_with_user_group_rbac(klass, scope, rbac_filters, find_opti
     end
   end
 
+  def self.group(user_or_group)
+    case user_or_group
+    when User
+      user_or_group.current_group
+    when MiqGroup
+      user_or_group
+    when NilClass
```

Comment:
I'd like to create tests cases and let them guide the design.  What are the test cases for the illustration above?  Or a better illustration?

So much of this stuff feels so intangible, I'd rather make decisions when we have some failing tests we need to make work.  We can decide where things belong then.